CHANGELOG.md

```diff
@@ -1,9 +1,7 @@
-# Changelog
+# Change Log
 
 All notable changes to this project will be documented in this file.
-
-The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
-and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
```

Comment:
Does conventional commit also enforce Semver?

Reply:
These changes are made by the automated changelog generator of lerna.
I can add these back at the end of the file if we want.

Reply:
Will Lerna overwrite this every time?

Reply:
No it just prepends the new format on top of the file.